### ISSUE TITLE
Filetype filter

### ISSUE THREAD

**Issue by flocke** [vauvenal5/yaga#65]
It would be nice to have an option to exclude certain filetypes from being shown/downloaded. My camera app usually saves the images as both JPG and as raw DNG files. Naturally the raw files are a lot bigger, so it would be nice if Yaga would only load and display the JPG files instead if wasting resources and bandwidth on the DNGs.

**Comment by vauvenal5** [comment]
Usually Yaga pulls only previews which are rather resources friendly, but I get your point.

**Comment by flocke** [comment]
Yeah, I noticed that it doesn't download the files later on as well. I was just confused because it shows a spinning circle for every DNG file that makes it look like it is loading forever. Just not displaying those circles for unsupported filetypes should be fine.

**Comment by vauvenal5** [comment]
Ah I see the issue. Does it download correctly if you open the file?

**Comment by flocke** [comment]
It downloads the file but it can't display it:
![Screenshot_20210122-185437_Yaga](https://user-images.githubusercontent.com/76665/105527176-a590e480-5ce3-11eb-845a-820d0751db2b.png)

And this is how the main view looks like with the DNG files next to each JPG image:
![Screenshot_20210122-185448_Yaga_1](https://user-images.githubusercontent.com/76665/105527253-bf322c00-5ce3-11eb-92a2-fd5a720127e5.png)

**Comment by vauvenal5** [comment]
Okay in this issue are basically three issues mixed up:
1. support for DNG format, moved to #66 
2. showing a default image when preview can not be load instead of infinity loading, moved to #67 
3. filtering file list, still open if only by file type or regex. For the third one I will leave this issue.
